### PR TITLE
Pocl: switch from deprecated master branch to main branch in git repo…

### PR DESCRIPTION
**Pull request by BOUDAOUD34**
https://github.com/pocl/pocl/pull/1275

### CODE REVIEW

File: var/spack/repos/builtin/packages/pocl/package.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Pocl(CMakePackage):
 
     license("MIT")
 
-    version("master", branch="master")
+    version("master", branch="main")
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/pocl/package.py]: Should we just call it `main`?

Comment (BOUDAOUD34) [reply var/spack/repos/builtin/packages/pocl/package.py]: Yes, why not

Comment (wdconinc) [reply var/spack/repos/builtin/packages/pocl/package.py]: Or both, with `master` deprecated. See e.g. https://github.com/spack/spack/blob/0619a8bd4f2238d87b22779b30b6b923d83eb846/var/spack/repos/builtin/packages/acts/package.py#L42-L43

Comment (BOUDAOUD34) [reply var/spack/repos/builtin/packages/pocl/package.py]: Done

Comment (wdconinc) [reply var/spack/repos/builtin/packages/pocl/package.py]: A lot of unrelated commits have snuck in. Can you remove them?

Comment (BOUDAOUD34) [reply var/spack/repos/builtin/packages/pocl/package.py]: I've just deleted them

**Review by alalazo — APPROVED** [review]

**Pull request merged**